imagersite/imager_images/models.py

```diff
@@ -0,0 +1,51 @@
+from django.db import models
+from django.contrib.auth.models import User
+from sorl.thumbnail import ImageField
+
+
+class Album(models.Model):
+    """Photo album model."""
+    user = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name='albums')
+    cover = models.ForeignKey(
+        'Photo',
+        on_delete=models.CASCADE,
```

Comment:
You might not want to have this as `CASCADE`. This means that when the `Photo` set as the cover is deleted, all the `Album`s it is set as the cover of will also be deleted. Since this is a `null=True` field, it might be better to set the field to null when the cover is deleted.